alembic/env.py

```diff
@@ -0,0 +1,79 @@
+from __future__ import with_statement
```

Comment:
Quoting George Silvis, III (2015-08-18 16:43:51)

>    Copyright header

Talked about this in person, but... Most of this is just the output of
`alembic init`? Don't know how that works; I only did some small tweaks
to get it to look at haas.cfg for some stuff. I wish I'd done two
commits so it was clear from the history what they were --- oh well.